frameworks/Java/armeria/src/main/java/hello/helpers/HttpHeadersHelper.java

```diff
@@ -0,0 +1,22 @@
+package hello.helpers;
+
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+import com.linecorp.armeria.common.HttpHeaderNames;
+import com.linecorp.armeria.common.HttpHeaders;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+
+public class HttpHeadersHelper {
+  public HttpHeaders getHttpHeader(MediaType mediaType) {
```

Comment:
This method doesn't depend on any internal state so it can be `static`:

```java
public static HttpHeaders getHttpHeader(MediaType mediaType) {
```

That way you don't need to create a new instance of `HttpHeadersHelper` in order to use the method.  Where you call it, it'll look like:
```java
HttpHeadersHelper.getHttpHeader(...)
```
rather than:
```java
new HttpHeadersHelper.getHttpHeader(...)
```